Jenkinsfile

```diff
@@ -0,0 +1,36 @@
+@Library('podTemplateLib')
+import net.santiment.utils.podTemplates
+
+properties([buildDiscarder(logRotator(artifactDaysToKeepStr: '30', artifactNumToKeepStr: '', daysToKeepStr: '30', numToKeepStr: ''))])
+
+slaveTemplates = new podTemplates()
+
+slaveTemplates.dockerTemplate { label ->
+  node(label) {
+    stage('Build') {
+      container('docker') {
+        def scmVars = checkout scm
+
```

Comment:
You can add a step for running the tests if you have any. You'll need `Dockerfile-test`(with your testing command)  , then build and run the image

Reply:
Yes, I will add them in the next version
Also I want to run tests for contracts